project.clj

```diff
@@ -27,7 +27,7 @@
 
   :min-lein-version "2.9.1"
 
-  :parent-project {:coords [puppetlabs/clj-parent "5.2.3"]
+  :parent-project {:coords [puppetlabs/clj-parent "5.2.11"]
                    :inherit [:managed-dependencies]}
 
   :dependencies [[org.clojure/clojure]
```

Comment:
there's a version for snakeyaml declared below, we should take that out in 7.x and include that in the mergeup